README.md

```diff
@@ -100,6 +100,68 @@ for (uint32_t i = 0; i < current_l2->processor_count; i++) {
 pthread_setaffinity_np(pthread_self(), sizeof(cpu_set_t), &cpu_set);
 ```
 
+## Use via pkg-config
+
+If you would like to automatically have your project's build environment use the appropriate compiler and linker build flags, [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) can greatly simplify things. It is the portable international _de facto_ standard for determining build flags. Here is an example:
```

Comment:
This reads like an ad for pkg-config, and it isn't appropriate here. The target audience for this section should be developers who already know what `pkg-config` is.

Reply:
Reworded and improved some of the notes.